### PR TITLE
Fixes #596 now susper works faster

### LINKED ISSUE #596: Speed up susper
the start page of susper is very slow.we need to speed up start page.

### PR DESCRIPTION
**Pull request by nikhilrayaprolu**
Fixes issue #596 

Changes: 

- enabled production mode

- shrinks vendor.js
- shrinks main.js
- enabled aot
- enabled service workers

Demo Link: https://nikhilrayaprolu.github.io/susper.com-1/

Screenshots for the change:

### CODE REVIEW

**Comment by mariobehling** [comment]
Please add a comparison chart of size of page and load times.

**Comment by nikhilrayaprolu** [comment]
Now:-
**vendor file statistics**:-
image
contentlength: 216764 content-encoding: gzip

**main.js bundle file**:
image
content length: 138361

**before**
**vendor file**:
image
content length: 709752
main.js bundle file:
content lenth: 56412
image


**all files will be initiated by service worker now:-**
image

**Comment by mariobehling** [comment]
Thanks, please find more ways to optimize the speed.

File: angular-cli.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
   "project": {
-    "version": "1.0.0-beta.24",
```
Comment (Marauderer97) [inline angular-cli.json]: Why has this been removed?

File: package.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
     "@angular/platform-browser": "4.1.3",
     "@angular/platform-browser-dynamic": "4.1.3",
     "@angular/router": "4.1.3",
+    "@angular/service-worker": "^1.0.0-beta.16",
```
Comment (Marauderer97) [inline package.json]: Whais is the significance of this?

File: package.json [hunk 3]
```diff
@@ -37,11 +38,11 @@
     "reselect": "^3.0.1",
     "rxjs": "5.4.0",
     "ts-helpers": "^1.1.2",
-    "zone.js": "^0.8.11",
```
Comment (Marauderer97) [inline package.json]: What is the significance of this re-arrangement?

File: angular-cli.json (hunk 1 above)
Comment (nikhilrayaprolu) [inline angular-cli.json]: that is not our project version, it's angular version.so removed it.

File: package.json (hunk 2 above)
Comment (nikhilrayaprolu) [inline package.json]: regarding service workers please read at https://developer.mozilla.org/en/docs/Web/API/Service_Worker_API or https://coryrylan.com/blog/fast-offline-angular-apps-with-service-workers
I will also write a blog post regarding that in susper.

File: package.json (hunk 3 above)
Comment (nikhilrayaprolu) [inline package.json]: It is auto rearrangement done when a new module is installed

Comment (AnshulMalik) [inline package.json]: Dependencies are listed alphabetically so that they are easy to find and organizable.

**Comment by mariobehling** [comment]
@nikhilrayaprolu Please follow up on best practices mentioned by @AnshulMalik in following PRs.
Thank you.

**Pull request merged**